CruCentralCoast/Events/EventsCVC.swift

```diff
@@ -9,43 +9,100 @@
 import UIKit
 
 class EventsCVC: UICollectionViewController {
+    var selectedCell: UIView?
+    var originFrame: CGRect?
     
     //some data for testing
-    var dataArray: [EventCellParameters] = [EventCellParameters(title: "Oasis", date: "March 17-26", location: "TBD", description: "ACM, the world's largest educational and scientific computing society, delivers resources that advance computing as a science and a profession. ACM provides the computing field's premier Digital Library and serves its members and the computing profession with leading-edge publications, conferences, and career resources."), EventCellParameters(title: "test2", date: "date", location: "location", description: "description"), EventCellParameters(title: "test3", date: "date", location: "location", description: "description"), EventCellParameters(title: "test4", date: "date", location: "location", description: "description"), EventCellParameters(title: "test5", date: "date", location: "location", description: "description")]
+    var dataArray = [Event]()
+    
+    var statusBarIsHidden: Bool = false {
+        didSet{
+            UIView.animate(withDuration: 0.25) { () -> Void in
+                self.setNeedsStatusBarAppearanceUpdate()
+            }
+        }
+    }
+    
+    override var prefersStatusBarHidden: Bool {
+        get {
+            return statusBarIsHidden
+        }
+    }
+    
+    override var preferredStatusBarUpdateAnimation: UIStatusBarAnimation {
+        return .slide
+    }
     
     override func viewDidLoad() {
```

Comment:
The code in this function could be spread out a bit more to improve readability. Specifically, an empty line could be added after "super.viewDidLoad()" and "registerCell()" and "layout.itemSize = ..."

Reply:
EventsCVC is not being used anymore. I will move it to a folder